Dockerfile

```diff
@@ -3,26 +3,20 @@ ARG NODE_IMAGE_VERSION=13.5.0-alpine
 FROM node:$NODE_IMAGE_VERSION as builder
 
 COPY package*.json ./
-
 RUN npm install
 
 COPY . .
 
 RUN npm run compile
-
-RUN npm run generateDocs
-
 RUN npm test
+RUN npm run generateDocs
 
 FROM node:$NODE_IMAGE_VERSION
 
 COPY --from=builder dist dist
-COPY package*.json ./
-COPY docs /docs
-
-RUN npm install --production
-
-COPY docs /docs
+COPY --from=builder node_modules node_modules
+COPY --from=builder package*.json ./
```

Comment:
I assume the wildcard is here for the lock file? Do we really need to copy the lock file? If so, is the wildcard really needed for it?

Reply:
Yes, the wildcard will include the lock file.
We need it to ensure the dependency tree is the same identical tree we use, regardless of intermediate dependency updates.

Reply:
You can explicitly copy package-lock.json and package.json in two steps

Reply:
Done ✅